src/components/utils/test/data_accessor_test.cc

```diff
@@ -52,17 +52,17 @@ TEST(DataAccessorTest, CreateDataAccessor) {
 TEST(DataAccessorTest, CreateDataAccessor_MutexIsLocked_CannotLockItAgain) {
   // arrange
   int test_value = 10;
-  sync_primitives::Lock testSet_lock_;
+  sync_primitives::Lock testSet_lock_(true);
```

Comment:
@OHerasym 
From test name it looks like it's NOT expected to have mutex locked twice.